index.php

```diff
@@ -96,7 +96,7 @@
 			<div id="search_content">
 				
 				<center><form action="search.php" method="post">
-					<input type="text" autocomplete="off" class="search" style = "border: 0.25vw solid #C2C2C2; background-color: #C2C2C2; font-family: 'Russo One', sans-serif;" placeholder="Search For Music..." name="search">
+					<input type="text" autocomplete="off" class="search" style = "border: 0.25vw solid #fff; font-family: 'Roboto', sans-serif;" placeholder="Search For Music..." name="search">

```

Comment:
Majority consists of applying "Roboto" as the heading font and changing the font colour to white from here and below.